test/utils/HubPoolUtils.ts

```diff
@@ -1,8 +1,24 @@
 // Propose and validate `numberOfBundles` bundles, each with random size block ranges. The block range size
 
+import { Contract, BigNumber } from "ethers";
 import { SpokePoolClientsByChain } from "../../src/interfaces";
-import { BigNumber, createRandomBytes32, toBN } from "../constants";
 import { MockHubPoolClient } from "../mocks/MockHubPoolClient";
+import {
+  SignerWithAddress,
+  createRandomBytes32,
+  toBN,
+  AcrossConfigStore,
+  getContractFactory,
+  utf8ToHex,
+  constants,
+  seedWallet,
+  MAX_L1_TOKENS_PER_POOL_REBALANCE_LEAF,
+  MAX_REFUNDS_PER_RELAYER_REFUND_LEAF,
+  getSampleRateModel,
+  TokenRolesEnum,
+} from "./index";
```

Comment:
Does this risk creating a circular dependency on imports?

Reply:
Is there a specific circular dep I created or are there many?

I was thinking that index should export everything out of utils while the individual utils files should load from index to get things from other utils files